packages/core/admin/admin/src/content-manager/pages/ListView/components/TableCells/Relations.tsx

```diff
@@ -43,113 +35,107 @@ const TypographyMaxWidth = styled(Typography)`
-  //                 <CellValue
-  //                   type={metadatas.mainField?.type ?? 'integer'}
-  //                   // @ts-expect-error – can't use a string to index the RelationResult object.
-  //                   value={metadatas.mainField?.name ? entry[metadatas.mainField.name] : entry.id}
-  //                 />
-  //               </TypographyMaxWidth>
-  //             </Menu.Item>
-  //           ))}
-
-  //           {data?.pagination && data?.pagination.total > 10 && (
-  //             <Menu.Item
-  //               aria-disabled
-  //               aria-label={formatMessage({
-  //                 id: getTranslation('ListViewTable.relation-more'),
-  //                 defaultMessage: 'This relation contains more entities than displayed',
-  //               })}
-  //             >
-  //               <Typography>…</Typography>
-  //             </Menu.Item>
-  //           )}
-  //         </>
-  //       )}
-  //     </Menu.Content>
-  //   </Menu.Root>
-  // );
+const RelationMultiple = ({ mainField, content, rowId, name }: RelationMultipleProps) => {
+  const { model } = useDoc();
+  const { formatMessage } = useIntl();
+  const { notifyStatus } = useNotifyAT();
+  const [isOpen, setIsOpen] = React.useState(false);
+
+  const [targetField] = name.split('.');
+
+  const { data, isLoading } = useGetRelationsQuery(
+    {
+      model,
+      id: rowId,
+      targetField,
+    },
+    {
+      skip: !isOpen,
+      refetchOnMountOrArgChange: true,
+    }
+  );
+
+  React.useEffect(() => {
+    if (data) {
+      notifyStatus(
+        formatMessage({
+          id: getTranslation('DynamicTable.relation-loaded'),
+          defaultMessage: 'Relations have been loaded',
+        })
+      );
+    }
+  }, [data, formatMessage, notifyStatus]);
+
+  return (
+    <Menu.Root onOpenChange={(isOpen) => setIsOpen(isOpen)}>
+      <MenuTrigger onClick={(e) => e.stopPropagation()}>
+        <Flex gap={1} wrap="nowrap">
+          <Badge>{content.count}</Badge>
+          {formatMessage(
+            {
+              id: 'content-manager.containers.list.items',
+              defaultMessage: '{number, plural, =0 {items} one {item} other {items}}',
+            },
+            { number: content.count }
+          )}
+        </Flex>
+      </MenuTrigger>
+      <Menu.Content>
+        {isLoading && (
+          <Menu.Item disabled>
+            <Loader small>
+              {formatMessage({
+                id: getTranslation('ListViewTable.relation-loading'),
+                defaultMessage: 'Relations are loading',
+              })}
+            </Loader>
+          </Menu.Item>
+        )}
+        {data?.results && (
+          <>
+            {data.results.map((entry) => (
+              <Menu.Item key={entry.documentId} disabled>
+                <TypographyMaxWidth ellipsis>
+                  {mainField && typeof entry[mainField] === 'string'
+                    ? (entry[mainField] as string)
```

Comment:
```suggestion
                    ? entry[mainField]
```

?

Reply:
I had TS issues, upon reflect, i can assert this better though.